README.md

```diff
@@ -1 +1,92 @@
-# sound_manager
+# autonomous_driving_sound_manager
```

Comment:
@MasaakiWatanabeTier4 
The README title has been changed from ad_sound_manager to autonomous_driving_sound_manager for clarity.